akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -88,12 +130,12 @@ private object TimerMessages {
 abstract class GraphStageLogic private[stream] (inCount: Int, outCount: Int) {
   import GraphInterpreter._
   import TimerMessages._
+  import GraphStageLogic._
 
   def this(shape: Shape) = this(shape.inlets.size, shape.outlets.size)
 
   private val keyToTimers = mutable.Map[Any, Timer]()
   private val timerIdGen = Iterator from 1
```

Comment:
I also changed this from an Iterator to a var in my other branch to reduce memory overhead.